include/ipr/interface

```diff
@@ -1558,11 +1583,11 @@ namespace ipr {
    // An expression of the form "static_cast<type>(expr)".
    struct Static_cast : Cast_expr<static_cast_cat> { };
 
-                                   // -- Qualification --
-   // A conversion that add cv-qualifiers
-   struct Qualification : Binary<Category<qualification_cat>, const Expr&, Type_qualifier> {
+                                // -- Widen --
+   // Derived to base class conversion
+   struct Widen : Binary<Category<widen_cat>, const Expr&, const Class&> {
       Arg1_type expr() const { return first(); }
```

Comment:
The third template argument should be `const Type&` not `const Class&`.

Reply:
See narrow, will update.